composer.json

```diff
@@ -32,47 +32,47 @@
         "symfony/asset": "self.version",
         "symfony/browser-kit": "self.version",
         "symfony/class-loader": "self.version",
-        "symfony/config": "2.99.99",
+        "symfony/config": "self.version",
         "symfony/console": "2.99.99",
```

Comment:
Still faking here until https://github.com/doctrine/doctrine2/pull/1310 is merged